test/client_test.js

```diff
@@ -57,26 +57,95 @@ describe('Client', () => {
         .get('/Patient/eb3271e1-ae1b-4644-9332-41e32c829486')
         .reply(200, () => readStreamFor('patient.json'));
 
-      const response = await this.fhirClient.read('Patient', 'eb3271e1-ae1b-4644-9332-41e32c829486');
+      const response = await this.fhirClient.read({ resourceType: 'Patient', identifier: 'eb3271e1-ae1b-4644-9332-41e32c829486' });
 
       expect(response.resourceType).to.equal('Patient');
       expect(response.id).to.equal('eb3271e1-ae1b-4644-9332-41e32c829486');
     });
 
-    it('responds to #read, throwing any errors that occur', async function () {
+    it('responds to #read, throwing any errors for a missing resource', async function () {
```

Comment:
I'd prefer the more general language here. While this test is specifically testing a 404 error, `read` throws on any error.